packages/components/src/components/my-position/test.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   
+    <title>mi-my-position test page</title>
+    <script type="module" src="/build/mi-components.esm.js"></script>
+    <script nomodule src="/build/mi-components.js"></script>
+    <script src="https://polite-terms-watch.loca.lt/build/index.js"></script>
+    <!-- <script src="https://app.mapsindoors.com/mapsindoors/js/sdk/4.23.0/mapsindoors-4.23.0.js.gz?apiKey=mapspeople"></script> -->
```

Comment:
Seems like a test setup 😄

Reply:
@ammapspeople I changed this to use the current version of the SDK but this has to be updated before merging